### DISCUSSION TITLE
Embed music in presenation

### DISCUSSION THREAD

**Discussion by miranyx** [marp-team/marp#486] [Q&A] [answered]
I am required to embed music to be more specific either an mp3 or wav file into the preseentation that plays the whole time how can I achieve it?

**Comment by yhatt** [comment, marked as answer]
That sounds like to be a rare use case, but you can add a music to the HTML presentation by using [the `<audio>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/audio) in your Markdown after enabling HTML tags in your Marp tool.

```markdown
---
marp: true
_footer: <audio src="https://free-loops.com/data/mp3/04/9e/b3ad7494a00424dc61921e3b341d.mp3" controls loop />
---

> Sound: Island Mallets 100BPM Instrument
> https://free-loops.com/8330-island-mallets-100bpm.html

---

...
```

This slide puts the audio player to the footer in the first slide. After once played, the viewer can listen the sound throughout the whole of presentation.

You also can try to order the sound to play automatically by adding `autoplay` attribute to the `<audio>` element, but please note that [the most of recent browsers will block autoplay without the explicit action by the user.](https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide#autoplay_and_autoplay_blocking)
